analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/RequestsWithExcessiveLengthBase.cs

```diff
@@ -21,7 +21,10 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.IO;
```

Comment:
Don't forget to update RSPEC for the rule to ship the `web.config` part with it.